### ISSUE TITLE
separate kiwi build requires

### ISSUE THREAD

**Issue by lnussel** [openSUSE/open-build-service#1528]
as suggested by @adrianschroeter https://github.com/openSUSE/kiwi/pull/541 should be solved on OBS level. The problem are kiwi VMX builds for SLE that need kernel-default-base instead of kernel-default. Both packages conflict with each other. The special kiwi-desc-vmxboot-requires package however requires  kernel-default, so when the kiwi file specifies  kernel-default-base OBS throws a package conflict even though the kernel packages are just meant to be put in the repo used by kiwi, they are not required to set up the build environment. So the conflict actually doesn't matter. Therefore OBS should ignore conflicts between packages that are just input for kiwi.

**Comment by mlschroe** [comment]
Nope, can't do that. Conflicts are also used the limit the packages providing some dependencies, I can't simply turn them off. I'll talk with Adrian.

**Comment by lnussel** [comment]
any update here? this blocks JeOS
